test/cardOrders.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const Fs = require('fs');
+const file = Fs.readFileSync(`${__dirname}/assets/card.pdf`);
+
+describe('cardOrders', () => {
+
+  describe('list', () => {
+
+    it('lists card orders given a card id', (done) => {
+      Lob.cards.create({
+        description: 'Test Card',
+        front: file,
+        back: file,
+        size: '2.125x3.375',
+      }, (err, res) => {
+        Lob.cardOrders.list(res.id, null, (err2, res2) => {
+          expect(res2.object).to.eql('list');
+          done();
+        });
+      });
+    });
+
+  });
+
+  describe('create', () => {
+
+    it('creates a card order', (done) => {
```

Comment:
this case exercises uploading a file. is it worth a test case exercising the path where we pass a url?